src/pydocstyle/cli.py

```diff
@@ -49,7 +49,8 @@ def run_pydocstyle():
 
     count = 0
     for error in errors:
-        sys.stdout.write('%s\n' % error)
+        if hasattr(error, 'code'):
```

Comment:
@farmersez why is `isinstance` better? I think this clearly reads as "print any error that has an error code associated with it".